### ISSUE TITLE
Failure to publish WASM when project source path contains blank spaces - Reopened from #26061

### ISSUE THREAD

**Issue by NonoSparc** [dotnet/sdk#27073]
All details are in post #26061.

It has been said that above issue should be solved with version 6.0.303 and onwards of .NET SDK. But no.

Upgraded VS 2022 to v 17.3.0. I now have both .NET SDK version 6.0.303 and 6.0.400 installed but none of them solve the issue.

Still have to rely on using the trick with global.json file in path above the project forcing to using an older version of SDK (6.0.203 in that matter).

Please advise.

**Comment by SantosVictorero** [comment]
Yes I agree, I upgraded VS 2022 to v17.3.0, removed the global.json with SDK(6.300) and the problem is still there: 

C:\Program Files\dotnet\sdk\6.0.400\Sdks\Microsoft.NET.Sdk.BlazorWebAssembly\targets\Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets(614,5): error MSB6006: "C:\Program Files\dotnet" exited with code 1.

**Comment by Sipowiz** [comment]
Same problem here. Mentioned update didn't resolve the issue. I'm using Azure DevOps Pipelines to publishing (.Net Core).

**Comment by mattiahj** [comment]
Yeah, I figured it could have been a regression in 400, but 303 does not work either. 300 is the last working version.

**Comment by SirQuillium** [comment]
Just wanted to make a comment here saying I updated to vs 17.3.0 and;

Unisntalled the sdk 303, installed 300

Ran the force script for the global.json "dotnet new globaljson --force --sdk-version 6.0.300" (cleaned and deleted bin and obj - as well as restarted VS)

Publish interface was prompting me that the publish failed(and that I should check the output), upon checking the output console I was now having a nuget package error: NU1605 - which I never had issues with before and haven't changed any packages. I even added the NetCore.Tagets package from the suggested fixes for that error code.

Whats funny is in the output it was also saying the publish succeeded - and upon checking the site - it was indeed updated and everything working. 

Publishing blazor WASM project to a Azure App Service - nothing fancy.
